libs/math/tests/unit/fixed_point/fixed_point.cpp

```diff
@@ -224,6 +224,104 @@ TEST(FixedPointTest, Conversion_32_32)
   EXPECT_EQ(fp64_t::DECIMAL_DIGITS, 9);
 }
 
+// TEST(FixedPointTest, Conversion_64_64)
```

Comment:
I'd suggest to delete or fix this test.

Reply:
it's actually removed, was accidentally added in this PR